tests/concat.rs

```diff
@@ -0,0 +1,148 @@
+use predicates::prelude::*;
+use snapbox::cmd::{cargo_bin, Command};
+use std::path::Path;
+
+pub mod common;
+use crate::common::*;
+
+#[path = "../src/io.rs"]
+pub mod io;
+
+use sketchlib::multisketch::MultiSketch;
+
+#[cfg(test)]
+
+mod tests {
+    use super::*;
+
+    #[test]
+    fn concat_competibility_test() {
```

Comment:
I actually think this entire test is redundant (and can be removed) because you test for failure with overlapping labels below

Reply:
This also means you can remove the two new files in `test_files_in`